apps/docs/src/docs/components/demo/PopoverVariant.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <!-- #region template -->
+  <div class="d-flex gap-2">
+    <BButton id="popover-variant">Button</BButton>
+    <BPopover
+      target="popover-variant"
+      title-class="text-bg-danger"
+      body-class="text-bg-danger"
```

Comment:
```suggestion
      class="text-bg-danger"
```

the body-class isn't that great with backgrounds, since bootstrap css doesn't contain the rounded borders for it, like it does for header. So using it makes the bottom non-rounded.